packages/angular/test/base/src/app/lazy/inputs/inputs.component.html

```diff
@@ -71,6 +71,16 @@
       <ion-note slot="end">{{input}}</ion-note>
     </ion-item>
 
+    <ion-item>
+      <ion-input-otp [(ngModel)]="inputOtp">Input OTP</ion-input-otp>
+      <ion-note slot="end">{{inputOtp}}</ion-note>
+    </ion-item>
+
+    <ion-item color="dark">
+      <ion-input-otp [(ngModel)]="inputOtp">Input OTP Mirror</ion-input-otp>
```

Comment:
Should this also mirror the backspace? The value updates when I enter a value but when I delete a value, the mirror doesn't reflect.